src/Stream.test.ts

```diff
@@ -81,6 +81,45 @@ describe('Stream', () => {
       await Fx.runAsync(test).promise
     })
   })
+
+  describe('fromAsyncIterable', () => { 
+    it('converts an async iterable to a stream', async () => {
+      const makeAsyncIterable = async function* () { 
+        for (let i = 0; i < 25; i++) {
+          yield i
+          await new Promise(resolve => setTimeout(resolve, 1))
```

Comment:
Do we need this delay for a specific reason?  Is this sufficient:

```suggestion
          yield Promise.resolve(i)
```

Reply:
No not really, I just wanted to add something "asynchronous", I'll try swapping it for yield Promise.resolve

Reply:
👌 Ultimately, either is fine with me, but if `yield Promise.resolve(i)` is sufficient, let's go with that.

Reply:
Updated https://github.com/briancavalier/fx/pull/5/commits/22f82d8b305c531f2fcc9486b5e01a8142de0aa3